shogun-boot/src/main/resources/db/migration/V1.0.0__Baseline.sql

```diff
@@ -53,41 +53,24 @@ CREATE TABLE files (
     CONSTRAINT files_unique_id UNIQUE (id)
 );
 
-CREATE TABLE roles (
-    id bigint NOT NULL,
-    created timestamp without time zone,
-    modified timestamp without time zone,
-    description text,
-    name text NOT NULL,
-    CONSTRAINT roles_pkey PRIMARY KEY (id),
-    CONSTRAINT roles_unique_id UNIQUE (id),
-    CONSTRAINT roles_unique_name UNIQUE (name)
-);
-
 CREATE TABLE users (
     id bigint NOT NULL,
     created timestamp without time zone,
     modified timestamp without time zone,
+    keycloak_id text NOT NULL,
     client_config jsonb,
     details jsonb,
-    email text NOT NULL,
-    enabled boolean,
-    password text,
-    username text NOT NULL,
     CONSTRAINT users_pkey PRIMARY KEY (id),
-    CONSTRAINT users_unique_email UNIQUE (email),
-    CONSTRAINT users_unique_id UNIQUE (id),
-    CONSTRAINT users_unique_username UNIQUE (username)
+    CONSTRAINT users_unique_id UNIQUE (id)
```

Comment:
You can simplify the unique and pk constraints by using `id bigserial` and annotating the column with
```
@Column
@GeneratedValue(strategy = GenerationType.IDENTITY)
@Id
```

Reply:
Herewith, sequential IDs would be generated automatically per table, wouldn't they?

Reply:
so i can remove the constraint lines?

Reply:
Yep. `id bigserial` (64 bit int) or `id serial` (32 bit int) will add the constraints, set the field as primary key and add a new sequence. This has the added benefit that objects in all tables start with id `1`.

Reply:
I'll change this in an follow up PR as this affects all ids.